manifests/deployment.yml

```diff
@@ -29,7 +29,7 @@ spec:
       serviceAccount: karydia
       containers:
       - name: karydia
-        image: karydia/karydia:0.1.0
+        image: eu.gcr.io/gardener-project/karydia/karydia
```

Comment:
Unfortunately, this is not the only place where we have to change the image. Here're some files which need to be adjusted as well (list may be incomplete):
- Makefile
- tests/e2e/framework/framework.go
- docs/devel/hotswap.md
- docs/install.md
- contrib/gardener/scripts/deploy-karydia-cp

Reply:
Changed image in files:
- Makefile
- docs/devel/hotswap.md
- docs/install.md
- scripts/generate-deployment-dev
- contrib/gardener/scripts/deploy-karydia-cp

Reply:
You changed the image at `tests/e2e/framework/framework.go` as well (just mentioning).